includes/wikia/models/DesignSystem/WdsExternalImage.class.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+class WdsExternalImage {
```

Comment:
It depends. Usually `get()` method gives you more flexibility to develop your classes. It's also easier to override when inheriting. In this case where use classes only for data representation then I think it's ok to skip getters.